coalib/parsing/LineParser.py

```diff
@@ -78,6 +79,16 @@ def _parse(self, line):
                      [(section_override, key), ...], value, to_append (True if
                      append delimiter is found else False), comment
         """
+        for separator in self.comment_separators:
+            if ((re.match('[^ ]' + separator, line)
+                    or re.match(separator + '[^ ]', line))
+                    and len(line) > 2):
```

Comment:
Why do we need to test the length of the line? If it matches either of the two regexes, then it's automatically at least 2 characters long.